Rubberduck.Inspections/Concrete/UnreachableCaseInspection/UCIValueExpressionEvaluator.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Rubberduck.Parsing.Grammar;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Rubberduck.Inspections.Concrete.UnreachableCaseInspection
+{
+    public interface IUCIValueExpressionEvaluator
+    {
+        IUCIValue Evaluate(IUCIValue LHS, IUCIValue RHS, string opSymbol);
+        IUCIValue Evaluate(IUCIValue LHS, string opSymbol);
+    }
+
+    public class UCIValueExpressionEvaluator : IUCIValueExpressionEvaluator
```

Comment:
This class seems to deals with numeric data types? If so, I think the name should reflect that. I was wondering why I wasn't seeing string operators such as `Like` in the list.